STAR-Delegation/draft-ietf-acme-star-delegation.md

```diff
@@ -909,6 +968,7 @@ This document adds the following entries to the ACME Directory Metadata Fields r
 | Field Name | Field Type | Reference |
 |------------|------------|-----------|
 | delegation-enabled | boolean | RFC XXXX |
+| allow-certificate-get | boolean | RFC XXXX |
```

Comment:
No, it's already there (from 8739).

Reply:
No, 8739 has it wrapped in the `auto-renewal` sub-object.  This would be at the top-level, to distinguish it from the STAR-specific setting.

Reply:
Nope, my comment applies to the one in the metadata, not in the order.

Reply:
STAR extends the directory object [like this](https://www.rfc-editor.org/rfc/rfc8739#section-3.2):
```
 meta
   + auto-renewal
      + allow-certificate-get
```

whereas here we'd be doing:
```
 meta
   + allow-certificate-get
```

Reply:
Ugh, you're right and I got it wrong. Sorry.

Reply:
done in 9b463fa